src/main/java/org/apache/sysml/api/mlcontext/MLContextConversionUtil.java

```diff
@@ -187,6 +189,38 @@ public static MatrixObject matrixBlockToMatrixObject(String variableName, Matrix
 	}
 
 	/**
+	 * Convert a {@code FrameBlock} to a {@code FrameObject}.
+	 * 
+	 * @param variableName
+	 *            name of the variable associated with the frame
+	 * @param frameBlock
+	 *            frame as a FrameBlock
+	 * @param matrixMetadata
+	 *            the matrix metadata
+	 * @return the {@code FrameBlock} converted to a {@code FrameObject}
+	 */
+	public static FrameObject frameBlockToframeObject(String variableName, FrameBlock frameBlock,
```

Comment:
Hi @asurve. Please update line 202 "frameBlockToframeObject" to "frameBlockToFrameObject". Thanks!